### ISSUE TITLE
Emoji captions

### ISSUE THREAD

**Issue by Noitarud** [dessalines/thumb-key#1022]
### Describe your suggested feature

I'd like some kind of caption for emoji, I don't understand some of these images… then I could search better too if #626.

### Other details

Maybe show captions for one line only, scroll to position. One line to be changed to two if captions expand line too far.

### Acknowledgements

- [X] I have written a short but informative title.
- [X] I have updated the app to **[the latest version](https://github.com/dessalines/thumb-key/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [X] I have searched the existing issues and this is a new one, **NOT** a  duplicate or related to another open issue.
- [X] This is a **single** feature request, in case of multiple feature  requests I will open a separate issue for each one (they can always link to each other if related)
- [X] This is not a question or a discussion, in which case I should have  gone to [lemmy.ml/c/thumbkey](https://lemmy.ml/c/thumbkey)
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements. 🤡
- [X] I will fill out all of the requested information in this form.

**Comment by dessalines** [comment]
This was caused by #1162 

@gitterrost4 All the other keyboards must have this issue then also, unless they have exceptions for certain characters.

**Comment by dessalines** [comment]
I'm not sure what you mean, these are emoji characters, they don't have captions with them. Could you show an example of how other keyboards do what you're saying?

**Comment by Noitarud** [comment]
i know of no precedents except on webpages like https://emojipedia.org/

as for "one line", I meant elaborate on all on the [top] line only to save cluttering with words (show more icons).

- I am using a tablet (which has screen room) so my line request may seem a little inappropriate for phone users, so,
- phone/tablet users could hold-press for the description of the one to appear beneath.

**Comment by Noitarud** [comment]
Yes the search feature is a good idea to have, but the motive was I didn't want to choose the wrong one when there may appear to be subtle differences in the image - eg the second one above - is it contentment or smug or blush or a little fevered/hot… or is it the "I'm not telling" kind of subtle grin?

**Comment by Noitarud** [comment]
In #1589 says the picker cannot be customised since it is an android control. This suggestion would entail making a new picker.
